### DISCUSSION TITLE
errors in updated version of forge

### DISCUSSION THREAD

**Discussion by coffeedrinker0725** [lllyasviel/stable-diffusion-webui-forge#409] [Q&A]
when running faceswaplad on forge, two errors appeared: FaceSwapLab - ERROR - Failed to post-process 'tuple' object has no attribute 'copy' and FaceSwapLab - ERROR - Failed to swap face in postprocess method : 'tuple' object has no attribute 'height'. The faceswaplad running on older version was  OK.

**Comment by coffeedrinker0725** [comment]
above error appear only in process of img2img. its Ok with txt2img.

**Comment by catboxanon** [comment]
FaceSwapLab is not yet fully supported. https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/274
